common/tests/functional/test_cbma/cbma_mdm_integration.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+
+MDM_AGENT_DIR="/opt/mesh_com/modules/sc-mesh-secure-deployment/src/nats"
+MDM_AGENT_CERTS="${MDM_AGENT_DIR}/certs"
+
+MDM_SERVER_DIR="~/mdm-server"
```

Comment:
Maybe nothing wrong here but in general best to avoid using tilde within variable contents as they are not always expanded, I don't remember in which case, maybe if you get executed with /bin/sh instead of Bash for example; safer to use $HOME instead 🙂 But then again that could end up being empty in some rare cases, which here would be making you write at the root of the file system 🤔

Reply:
Done!

Using in the example "\$HOME" instead of "$HOME" as it should expand in the SSH server :label: